### DISCUSSION TITLE
Changing validator's response format

### DISCUSSION THREAD

**Discussion by rbgfloripa** [adonisjs/core#2600] [Help]
Is there any way to change the validator's default response?

From this
```Javascript
{
  "api-key": [
    "api-key header is required"
  ]
}
```

To this
```Javascript
{
  "status": "ERROR",
  "messages": [
    "api-key header is required"
  ],
  "content": null
}
```

TIA

Rafael

**Comment by thetutlage** [comment]
Hello @rbgfloripa 

Please checkout the error formatters doc https://docs.adonisjs.com/guides/validator/error-reporters
